test/issues/462/test.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const { after, before, describe, it } = require('node:test')
+const path = require('node:path')
+const Fastify = require('fastify')
+const autoLoad = require('../../../')
+const assert = require('node:assert')
+
+describe('Issue 462 tests', function () {
```

Comment:
A more descriptive title would be better


```suggestion
describe('ignore invalid and/or empty .js and .mjs files when using esm', function () {
```

Reply:
That's fine I think, if we need more context, we can read the issue.
Eventually we can open a PR to update the existing `describe` test files (which aren't descriptive either).

Reply:
I added a more descriptive title for the test case
```ts
describe('Issue 462: ignore invalid and/or empty .js and .mjs files when using esm', function () {
```